### PR TITLE
UP-34: Add LEDGERS_FREEZE and GET_FROZEN_LEDGERS transactions

### PR DESCRIPTION
**Pull request by Toktar**

### CODE REVIEW

**Comment by askolesov** [comment]
(ci) ok to test

**Review by askolesov — APPROVED** [review]
Cool

File: indy_node/server/request_handlers/config_req_handlers/ledgers_freeze_handler.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class LedgersFreezeHandler(PLedgersFreezeHandler):
+
+    def __init__(self, database_manager: DatabaseManager,
+                 write_req_validator: WriteRequestValidator):
+        super().__init__(database_manager)
+        self.write_req_validator = write_req_validator
+
+    def additional_dynamic_validation(self, request: Request, req_pp_time: Optional[int]):
+        pass
```
[the remark this answers is not in the document]
Comment (Toktar) [reply indy_node/server/request_handlers/config_req_handlers/ledgers_freeze_handler.py:19]: The super implementation is an abstract method. So, I guess we can't call super implementation or omit this override.

[the remark this answers is not in the document]
Comment (askolesov) [reply indy_node/server/request_handlers/config_req_handlers/ledgers_freeze_handler.py:19]: I thought super implementation here it the following:
https://github.com/Toktar/indy-plenum/blob/0448f51e6c7f30d2d8e8f156002ae4b8695ce00d/plenum/server/request_handlers/ledgers_freeze/ledgers_freeze_handler.py#L28
Am I wrong?

[the remark this answers is not in the document]
Comment (Toktar) [reply indy_node/server/request_handlers/config_req_handlers/ledgers_freeze_handler.py:19]: Oh, you are absolutely right, thanks! My mistake, sorry. I'll fix it.

[the remark this answers is not in the document]
Comment (askolesov) [reply indy_node/server/request_handlers/config_req_handlers/ledgers_freeze_handler.py]: Thanks!

**Review by askolesov — APPROVED** [review]

**Review by skhoroshavin — APPROVED** [review]

**Pull request merged**